{
  "description": "linux_auth_failed_test_case",
  "test_specification": {
    "event": {
      "original_string": "<34>1 2022-11-09T22:14:15.003Z spm-test su - - - 'su root' failed for test on /var/log",
      "syslog_hostname": "spm-test",
      "syslog_appname": "su",
      "syslog_timestamp": "2022-11-09T22:14:15.003Z",
      "syslog_facility": 4,
      "syslog_msg": "'su root' failed for test on /var/log",
      "syslog_priority": 34,
      "source_type": "linux_syslog",
      "syslog_version": 1,
      "syslog_severity": 2,
      "timestamp": 1667979913663,
      "enriched_hostname": "root-user"
    }
  },
  "assertions": [
    {
      "assertion_type": "path_and_value_matches",
      "json_path": "$.outputEvents[0].syslog_appname",
      "expected_pattern": "su",
      "negated_pattern": false,
      "active": true
    }
  ],
  "author": "siembol",
  "version": 3,
  "test_case_name": "linux_auth_failed_test_case",
  "config_name": "linux_syslog_failed_auth"
}